### PR TITLE
Create acosh entry and test all changes

### LINKED ISSUE #953: [Term Entry] C++ Math Functions: `acosh()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `acosh()` function.

- In the `terms/` folder, make a new folder called `acosh`
- Inside the `acosh` folder, create a new markdown file called `acosh.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ddiegow**
### Description

Create acosh() entry. Closes #953 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@ddiegow, this looks good for a second review.

File: content/cpp/concepts/math-functions/terms/acosh/acosh.md
Comment (KTom101) [inline content/cpp/concepts/math-functions/terms/acosh/acosh.md:22]:
```suggestion
The `n` parameter can be a floating-point value (e.g., a `long` or `double`) or an integral type (e.g., an `int` or `bool`). `n` is mandatory and must be greater or equal to 1. Values less than 1 will result in a domain error.
```

File: content/cpp/concepts/math-functions/terms/acosh/acosh.md
Comment (KTom101) [inline content/cpp/concepts/math-functions/terms/acosh/acosh.md:40]:
```suggestion
This results in the following output:
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@ddiegow I've added changes to this entry for you to take a look at.

**Review by KTom101 — APPROVED** [review]
@ddiegow Looks good to me 👍


@SSwiniarski any additional changes you want to add before merging?

**Comment by SSwiniarski** [comment]
@KTom101, nope, everything looks good to me.

**Comment by KTom101** [comment]
Hi @ddiegow thank you for your contribution! You can see the entry is live here: https://www.codecademy.com/resources/docs/cpp/math-functions/acosh

I've noticed that your username has not been added as a contributor. May I ask if you:

                1.  Added your GitHub username to your Codecademy profile;
Screen Shot 2022-12-17 at 5 57 07 PM

                2. Log out and then log back in on Codecademy using your GitHub login information.
Screen Shot 2022-12-17 at 6 00 50 PM
n Codecademy using your GitHub profile

**Pull request merged**
